kube/egressservices/egressservices.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Package egressservices contains shared types for exposing tailnet services to
```

Comment:
These configs will only ever be created by the Kubernetes operator, not users directly